frontend/openchat-client/src/openchat.ts

```diff
@@ -1694,6 +1698,14 @@ export class OpenChat extends EventTarget {
         const userIds = this.userIdsFromEvents(resp.events);
         await this.updateUserStore(chatId, userIds);
 
+        if (this._liveState.selectedThreadRootMessageIndex !== undefined) {
```

Comment:
Do we need to check that the selected thread hasn't changed?

Reply:
yes probably